modules/apps/frontend-theme/frontend-theme-cms/src/css/_clay_custom.scss

```diff
@@ -0,0 +1,319 @@
+$label-primary-close: () !default;
```

Comment:
@marcoscv-work just a question why all the variables here are in `_clay_custom.scss` instead of `_clay_variables.scss`

Reply:
@marcoscv-work I think it would be better in _clay_variables.scss with the _globals.scss import.